internal/cmd/beta/network-area/route/update/update.go

```diff
@@ -0,0 +1,166 @@
+package update
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	iaasUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+)
+
+const (
+	routeIdArg = "ROUTE_ID"
+
+	organizationIdFlag = "organization-id"
+	networkAreaIdFlag  = "network-area-id"
+
+	labelFlag = "label"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	OrganizationId *string
+	NetworkAreaId  *string
+	RouteId        string
+	Labels         *map[string]string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update",
+		Short: "Updates a static route in a STACKIT Network Area (SNA)",
+		Long: fmt.Sprintf("%s\n%s\n",
+			"Updates a static route in a STACKIT Network Area (SNA).",
+			"This command is currently asynchonous only due to limitations in the waiting functionality of the SDK. This will be updated in a future release.",
+		),
+		Args: args.SingleArg(routeIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(
+				`Updates the label(s) of a static route with ID "xxx" in a STACKIT Network Area with ID "yyy" in organization with ID "zzz"`,
+				"$ stackit beta network-area route update xxx --label key=value,foo=bar --organization-id yyy --network-area-id zzz",
```

Comment:
Since we have the flag in the singular form:
```suggestion
				"$ stackit beta network-area route update xxx --label key=value --label foo=bar --organization-id yyy --network-area-id zzz",
```

Reply:
Alternatively, and I think this could even make it more clear, the flag could have the name in the plural `labels`

Reply:
Changed to to `labels`